src/.vscode/settings.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Where are these changes coming from? They are not related to CDFJ+, are they?

Reply:
No - they come from the VS Code IDE - I though I had reverted them, but it looks like it didn't work.